doc/manuals/orion-api.md

```diff
@@ -1128,9 +1128,44 @@ Content-Length: 31
 The temperature is 23.4 degrees
 ```
 
-Some considerations to take into account:
+As alternative to `payload` field in `httpCustom` or `mqttCustom`, the `json` field can be
+used to generate JSON-based payloads. For instance:
 
-* It is the NGSIv2 client's responsability to ensure that after substitution, the notification is a
+```
+"httpCustom": {
+   ...
+   "json": {
+     "t": "${temperature}",
+     "h": [ "${humidityMin}", "${humidityMax}" ],
+     "v": 4
+   }
+}
+```
+
+Some notes to take into account when using `json`instead of `payload`:
+
+* The value of the `json` field must be an array or object. Although a simple string or number is
+  also a valid JSON, these cases are not supported.
+* The macro replacement logic works the same way than in `payload` case, with the following
+  considerations:
+  * It cannot be used in the key part of JSON objects, i.e. `"${key}": 10` will not work
+  * The value of the JSON object or JSON array item in which the macro is used has to match
+    exactly with the macro expression. Thus, `"t": "${temperature}"` works, but
+    `"t": "the temperature is ${temperature}"` or `"h": "humidity ranges from ${humidityMin} to ${humidityMax}"`
+    will not work
+  * It takes into account the nature of the attribute value to be replaced. For instance,
+    `"t": "${temperature}"` resolves to `"t": 10` if temperature attribute is a number or to
+    `"t": "10"` if `temperature` attribute is a string.
+  * If the attribute doesn't exist in the entity, then `null` value is used
+* URL automatic decoding applied to `payload` and `headers` fields (described
```

Comment:
The link is no longer valid when https://github.com/telefonicaid/fiware-orion/pull/4181#discussion_r935498892 gets addressed. Check and fix after that.